woocommerce/woocommerce-gutenberg-products-block
# [Store Customization] Update the "Footer with 3 Menus"

## What
This PR
- Increases the padding around the pattern.
- Increases the spacing between the 3 menus.
- Sets the image width to 60px width.
- Shows "Powered by Wordpress..." in just one line.
- Add bold to the site title.
- Make the search bar fill the space.

## Why

Fixes https://github.com/woocommerce/woocommerce-blocks/issues/11367


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Insert the `Footer with 3 menus` pattern.
3. Check that it looks like the design.


## Screenshots or screencast



| Desktop | 
| ------ | 
| <img width="1397" alt="Screenshot 2023-10-23 at 16 32 48" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/747643d2-7548-41d9-8ce1-8f91bcd1c171">|

| Mobile | 
| ------ | 
|<img width="334" alt="CleanShot 2023-10-23 at 16 33 30@2x" src="https://github.com/woocommerce/woocommerce-blocks/assets/186112/1d94f1ed-dad7-4b9b-ad9c-432acb16c2c2"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Improve the "Footer with 3 Menus" pattern.
